cluster/kubevirtci.sh

```diff
@@ -0,0 +1,22 @@
+export KUBEVIRT_PROVIDER=${KUBEVIRT_PROVIDER:-'k8s-1.17.0'}
+
+KUBEVIRTCI_VERSION='0c3911794ad2b79a61a4bc7462c236251a73866f'
+KUBEVIRTCI_PATH="${PWD}/_kubevirtci"
+
+function kubevirtci::install() {
+    if [ ! -d ${KUBEVIRTCI_PATH} ]; then
```

Comment:
I would love to check commit id here :-/

Reply:
Good point! Lemme try to implement it.